includes/class-gp-openai-review.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+class GP_OpenAI_Review {
+	/**
+	 * The OpenAI key.
+	 *
+	 * @var string
+	 */
+	private static $gp_openai_key = '';
+
+	/**
+	 * Get suggestions from OpenAI (ChatGPT).
+	 *
+	 * @param string $original_singular The singular from the original string.
+	 * @param string $translation       The translation.
+	 * @param string $locale            The locale.
+	 * @param string $glossary_query   The prompt generated to include glossary for the locale.
+	 *
+	 * @return array
+	 */
+	public static function get_openai_review( $original_singular, $translation, $locale, $glossary_query, $is_retry ): array {
+		$openai_query = '';
+		$openai_key   = apply_filters( 'gp_get_openai_key', self::$gp_openai_key );
+
+		if ( empty( trim( $openai_key ) ) ) {
+			return array();
+		}
+		$openai_temperature = 0;
+
+		$gp_locale     = GP_Locales::by_field( 'slug', $locale );
+		$openai_query .= 'For the english text  "' . $original_singular . '", is "' . $translation . '" a correct translation in ' . $gp_locale->english_name . '?';
+		$openai_query  = ( $is_retry ) ? 'Are you sure that ' . $openai_query : $openai_query;
```

Comment:
```suggestion
```

I'd just remove this. If you run into a OpenAI fluke (for example because of too many requests per minute) and you received a "Oops! No response from ChatGPT" the retry will say something along the lines of "I apologize for the mistake. " which is not accurate because there was no mistake.